cost-analyzer/values.yaml

```diff
@@ -22,6 +22,22 @@ global:
     proxy: true # If true, the kubecost frontend will route to your grafana through its service endpoint
 #    fqdn: cost-analyzer-grafana.default.svc
 
+  # Google Cloud Managed Service for Prometheus
+  gmp:
+  # Remember to set up these parameters when install the Kubecost Helm chart with `global.gmp.enabled=true` if you want to use GMP self-deployed collection (Recommended) to ultilize Kubecost scrape configs. 
+  # Learn more at https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-unmanaged
+  # --set prometheus.server.image.repository="gke.gcr.io/prometheus-engine/prometheus" \
+  # --set prometheus.server.image.tag="v2.35.0-gmp.2-gke.0"
```

Comment:
@linhlam-kc Is the intention here to recommend using a specific Prometheus image if `global.gmp.enabled=true`? If so, could we reword this for some added clarity? Maybe something like:

```
# If enabling GMP, it is also recommended to utilize Google's distribution of Prometheus.
# Learn more at https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-unmanaged
# --set prometheus.server.image.repository="gke.gcr.io/prometheus-engine/prometheus" \
# --set prometheus.server.image.tag="v2.35.0-gmp.2-gke.0"
```

Reply:
@thomasvn Yes, it is required to use GMP provided Prom image. Thank you for your feedback, I will update the description in the next commit.

Reply:
Updated in the latest commit.